connector/MongoDBAtlasODBC.pq

```diff
@@ -8,7 +8,7 @@ MongoDBAtlasODBCType = type function(
     mongodbUri as (type text meta[
         Documentation.FieldCaption = "MongoDB URI",
         Documentation.FieldDescription = "MongoDB connection URI",
-        Documentation.SampleValues = {"mongodb://hostname:port/admin"}
+        Documentation.SampleValues = {"mongodb://hostname:port/admin?ssl=true"}

```

Comment:
do we also want to have `authSource=something` so that it's in line with the proposed changes for the jdbc driver?

Reply:
Would that be instead of the `admin` value, such as `mongodb://hostname:port/?ssl=true&authSource=admin` ?

Reply:
I don't think so! I'm not sure, but I think we want the modal to look the same when it hits the UI. So the connection shown to users would be `mongodb://hostname:port?ssl=true&authSource=admin` and then the database is included from the info field, if I'm understanding it correctly.

Reply:
I updated the connection value to have the `authSource=admin` and removed the `admin` in the default db spot.  Yes there will be a `database` text field in the UI for the user to input the value.